corehq/apps/accounting/models.py

```diff
@@ -1965,12 +1966,8 @@ class Meta(object):
     _pdf = None
 
     @property
-    def recipients(self):
-        return self.emailed_to.split(',') if self.emailed_to else []
-
-    @recipients.setter
-    def recipients(self, emails):
-        self.emailed_to = ','.join(emails)
+    def emailed_to(self):
```

Comment:
I tested http://localhost:8000/hq/admin/api/global/billing_record/ and saw that the field emailed_to was not appearing.  There should be a field following the format in https://www.commcarehq.org/hq/admin/api/global/billing_record/

I think the solution is to add a field to BillingRecordResource like so: https://github.com/dimagi/commcare-hq/blob/5b34cca13912b21581302521a725a2bd809ee86e/corehq/apps/api/accounting.py#L134

and a method dehydrate_emailed_to like so: https://github.com/dimagi/commcare-hq/blob/5b34cca13912b21581302521a725a2bd809ee86e/corehq/apps/api/accounting.py#L142-L143.

Then emailed_to can be removed from fields, and this function can be dropped.